### PR TITLE
Update react native swiper

### LINKED ISSUE #1915: Cannot update react-native-swiper further than 1.5.10
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a developer, I want to be able to update the react-native-swiper to new versions in order to benefit from future performance improvements/bug fixing without breaking the application
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: 
Currently using a version of the react-native-swiper more recent than 1.5.10 gets you a white screen when you change to another tab in the main view after logging in.

Main screen works
screenshot_1506100359

Then tab gives this
screenshot_1506100363

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

Developer side:
- update react-native-swipper to 1.5.11 in package.json

Tester side:
- Open Status
- Log in
- Go to another tab
- Should show the tab and not a white screen

### Solution
[comment]: # (Please summarise the solution and provide a task list on what needs to be fixed.)
*Summary*: 

- [ ] reproduce on iOS to see if there is similar problem and maybe more detailed logs on the error

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: develop
[comment]: # (Android or iOS.)
* Operating System: Android

#### Logs
```
09-22 19:12:39.253  3388  3432 D ReactNativeJS: DEBUG [status-im.utils.handlers:22] - Handling re-frame event:  :navigate-to-tab
09-22 19:12:39.267  3388  3432 D ReactNativeJS: DEBUG [status-im.utils.handlers:22] - Handling re-frame event:  :update-wallet
09-22 19:12:39.282  3388  3432 D ReactNativeJS: DEBUG [status-im.utils.handlers:22] - Handling re-frame event:  :update-balance-fail
09-22 19:12:39.282  3388  3432 D ReactNativeJS: DEBUG [status-im.ui.screens.wallet.events:114] - Unable to get balance:  Error: can't fetch trie key 7f72b9a54d0cc314b0e2e584a4cb93e08e1380af86f2ca85ee325346fb8df8f6: no suitable peers available
09-22 19:12:39.343  3388  3432 D ReactNativeJS: DEBUG [status-im.utils.handlers:22] - Handling re-frame event:  :navigate-to-tab
09-22 19:12:39.403  3388  3388 W ViewPager: Requested offscreen page limit 0 too small; defaulting to 1
```

### LINKED ISSUE #2020: Last 2 transactions on History tab are shown outside of visible area after switching from Unsigned to History
### Description

*Type*: Bug

*Summary*: Wallet-> Transactions: It's possible to scroll list of transactions on the History tab to the very end and see all transactions. After switching to Unsigned tab and back to History tab: last 2 transactions are not shown in the visible area even if scroll to the very end. 

before moving to Unsigned
img_2346

After returning from Unsigned:
img_2345

#### Expected behavior
All transactions are shown 
#### Actual behavior
Last 2 transactions are hidden and user can't see them even if scroll down to the end

### Reproduction
from 00:28 https://app.testfairy.com/projects/4803590-status/builds/6722412/sessions/7/?accessToken=gL6g/xWA1JH2-zXfkCoRbQIhIgw

You need to have list of transactions in History that you know and it's not too big, e.g. a bit more than fits on 1 dcreen
- Open Status
- Open Wallet -> Transactions -> History
- Scroll history to the end to see the last transaction and remember it
- tap on Unsigned tab
- Tap on History tab
- scroll transactions to the end

### Additional Information
* Status version: develop
* Operating System: 
Real device iPhone 6s, iOs 10.3.1

### PR DESCRIPTION
**Pull request by yenda**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixes #1915
check if fixes #2020, write issue on react-native-swiper tracker if not

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
Refactor main-tabs so that react-native-swiper can be updated

### Steps to test:
- check that tabs swiping is working and tabs can be changed from the tab bar

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready

### CODE REVIEW

File: src/status_im/ui/screens/main_tabs/views.cljs [hunk 1]
```diff
@@ -1,114 +1,57 @@
...
+     (let [icon (if active? icon-active icon-inactive)]
+       [react/view
+        [vector-icons/icon icon (tabs.styles/tab-icon active?)]])
+     [react/view
+      [react/text {:style (tabs.styles/tab-title active?)}
+       title]]]))
+
+(def tabs-list*
+  [{:view-id :wallet
+    :content {:title (i18n/label :t/wallet)
```
Comment (x-goose-x) [inline src/status_im/ui/screens/main_tabs/views.cljs]: Map alignment

**Review by x-goose-x — APPROVED** [review]

**Review by rasom — APPROVED** [review]

File: src/status_im/ui/screens/wallet/transactions/styles.cljs [hunk 2]
```diff
@@ -17,14 +17,8 @@
...
-(def main-section
-  {:flex             1
-   :background-color styles/color-white})
-
-(def tab-height (if platform/ios? 51 55))
-
-(def tabs-container
-  {:flexDirection :row})
+(def tabs-height (if platform/ios? 52 56))
+(def tab-height (dec tabs-height))
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/transactions/styles.cljs]: Is this duplicated from `tabs/styles` ?

**Review by jeluard — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by yenda** [comment]
Let me squash the first two commits before merging

**Comment by annadanchenko** [comment]
#1915 and #2020 are ok both Android and iOS

**Pull request merged**